src/app/zones/locales/en-us/index.yaml

```diff
@@ -1,29 +1,47 @@
 zone-cps:
   routes:
     item:
-      title: "{name} Zone CP"
-      breadcrumbs: Zone CPs
+      title: "{name} Control Plane"
+      breadcrumbs: Control Planes
       config:
         no-subscriptions: This zone has no subscriptions
     items:
-      title: Zone CPs
-      breadcrumbs: Zone CPs
+      title: Control Planes
+      breadcrumbs: Control Planes
+      headers:
+        name: 'Name'
+        zonecpversion: 'Zone CP Version'
+        storetype: 'Storage type'
```

Comment:
Why can't these just be `version` and `storage`? What am I missing here?

Reply:
It’s not important that they are those particular values.

Reply:
Ok then if its not important to you can you change it please?

`version` and `storage` would be ideal (unless I'm missing something), partly so we aren't namespacing with `zonecp` twice. We know its a zonecp already.